proofOC/bookRoom.py

```diff
@@ -80,14 +87,24 @@ def transformData(self, timeS, roomNum, date):
             return None
         
 
-    def bookRoom(self, roomSize, timeS, roomNum, date):
+    def bookRoom(self, roomSize, timeS, roomNum, date, repoUrl):
 
         roomSize = roomSize.strip().lower()
         dateString = self.transformData(timeS, roomNum, date)
         if(dateString==None or (roomSize!="small" and roomSize!="large")):
             print('Incorrect Entry/Format')
             return False
 
+        # save booking to file
+        repo = RepoCommunicator(repoUrl, self.repoPath)
+
+        bookings = BookingBuilder(self.userN, self.passW)
+        bookings.addBooking(dateString, roomSize)
+
+        repo.addFile(bookings.fileName)
+        repo.pushData()
+
+
```

Comment:
It works! the only issue is that bookRoom.py runs selenium so it tries to book the room. Since bookRoom.py now just stores reservations we can get rid of that code.

Reply:
oop sorry I highlighted the wrong code - I'm talking about lines 108 to 148

Reply:
I opted not to remove the booking process until we can verify everything works through github actions. this way we still have something working e2e for tomorrow, and we don't lose some functionality while working on the rest.

I think we keep it for now, and remove it once we know the new workflow consistently works :)